accessible/appview.component.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * AccessibleBlockly
+ *
+ * Copyright 2016 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Angular2 Component that details how the AccessibleBlockly
+ * app is rendered on the page.
+ * @author madeeha@google.com (Madeeha Ghori)
+ */
+blocklyApp.workspace = blocklyApp.workspace || new Blockly.Workspace();
```

Comment:
Can you explain (perhaps in a comment) why you need the guard against blocklyApp.workspace being set here? Where else is blocklyApp.workspace set?